mypy/fastparse.py

```diff
@@ -294,11 +294,50 @@ def translate_expr_list(self, l: Sequence[AST]) -> List[Expression]:
             res.append(exp)
         return res
 
-    def translate_stmt_list(self, l: Sequence[AST]) -> List[Statement]:
+    def get_line(self, n: ast3.stmt) -> int:
```

Comment:
Maybe rename to `get_lineno`? Without context, the name `get_line` vaguely makes me think that it would get the source code line somehow.